api/server/call_logs.go

```diff
@@ -27,16 +28,16 @@ func (s *Server) handleCallLogGet(c *gin.Context) {
 		return
 	}
 
-	var b bytes.Buffer
-	b.ReadFrom(logReader)
-
 	mimeTypes, _ := c.Request.Header["Accept"]
 
 	for _, mimeType := range mimeTypes {
 		switch mimeType {
 		case "text/plain":
-			c.String(http.StatusOK, b.String())
+			io.Copy(c.Writer, logReader)
+			c.Status(http.StatusOK)
```

Comment:
the status code is already written at this point, can either remove this or move it before `io.Copy`, either works.